apps/dashboard/src/components/workflow-editor/steps/configure-step-form.tsx

```diff
@@ -335,27 +316,7 @@ export const ConfigureStepForm = (props: ConfigureStepFormProps) => {
             </>
           )}
 
-          {isStepConditionsEnabled && (
-            <>
-              <SidebarContent>
-                <Link to={'./conditions'} relative="path" state={{ stepType: step.type }}>
-                  <Button
-                    variant="secondary"
-                    mode="outline"
-                    className="flex w-full justify-start gap-1.5 text-xs font-medium"
-                  >
-                    <RiGuideFill className="h-4 w-4 text-neutral-600" />
-                    Skip Conditions
-                    <span className="ml-auto flex items-center gap-0.5">
-                      <span>{conditionsCount}</span>
-                      <RiArrowRightSLine className="ml-auto h-4 w-4 text-neutral-600" />
-                    </span>
-                  </Button>
-                </Link>
-              </SidebarContent>
-              <Separator />
-            </>
-          )}
```

Comment:
This component has already 400 lines, additionally I would need to add more logic like `uiSchema` and `skip` check just for the button, so I decided to move it to a separate component, its contained and easier to read. 

The `<Separator/>` issue is solved by a `inSidebar` flag, not ideal but arguably better than being limited to having everything in one component.